src/configuration/CertificateDetailsModal.tsx

```diff
@@ -0,0 +1,358 @@
+            <code>{formatCertSubject(crt)}</code>
+          </SplitItem>
+        </Split>
+        <Split hasGutter key={'subjectHash' + order}>
+          <SplitItem>
+            <b>SubjectHash:</b>
+          </SplitItem>
+          <SplitItem isFilled>
+            <code>{crt.subject.hash}</code>
+          </SplitItem>
+        </Split>
+        <Split hasGutter key={'signatureOid' + order}>
+          <SplitItem>
+            <b>SignatureOid:</b>
+          </SplitItem>
+          <SplitItem isFilled>
+            <code>{crt.signatureOid}</code>
+          </SplitItem>
+        </Split>
+        <Split hasGutter key={'split-signature'}>
+          <SplitItem key={'splititem-signatue-key'}>
+            <b>Signature:</b>
+          </SplitItem>
+          <SplitItem>
+            <CodeBlock
+              key={'key-codeblock-signature'}
+              id={'id-codeblock-signature'}
+              width={50}
+            >
+              <CodeBlockCode id="signature-code-content">
+                {formatSignature(crt.signature)}
+              </CodeBlockCode>
+            </CodeBlock>
+          </SplitItem>
+        </Split>
+        <Split hasGutter key={'siginfo' + order}>
+          <SplitItem>
+            <b>Siginfo:</b>
+          </SplitItem>
+        </Split>
+        <Split hasGutter key={'siginfo.algorithmOid' + order}>
+          <SplitItem></SplitItem>
+          <SplitItem>
+            <b>AlgorithmOid:</b>
+          </SplitItem>
+          <SplitItem isFilled>
+            <code>{crt.siginfo.algorithmOid}</code>
+          </SplitItem>
+        </Split>
+        <Split hasGutter key={'siginfo.parameters' + order}>
+          <SplitItem></SplitItem>
+          <SplitItem>
+            <b>Parameters:</b>
+          </SplitItem>
+          <SplitItem isFilled>
+            <code>{JSON.stringify(crt.siginfo.parameters)}</code>
+          </SplitItem>
+        </Split>
+
+        <Split hasGutter key={'extensions' + order}>
+          <SplitItem>
+            <b>Extensions({crt.extensions?.length})</b>:
+          </SplitItem>
+        </Split>
+        {getExtensionsList(crt, order)}
+
+        <Split hasGutter key={'split-publickey'}>
+          <SplitItem key={'splititem-publickey-key'}>
+            <b>{'PublicKey Info:'}</b>
+          </SplitItem>
+          <SplitItem>
+            <CodeBlock
+              key={'key-codeblock-publickey'}
+              id={'id-codeblock-publickey'}
+              width={50}
+            >
+              <CodeBlockCode id="publickey-code-content">
+                {formatPublicKey(crt.publicKey)}
+              </CodeBlockCode>
+            </CodeBlock>
+          </SplitItem>
+        </Split>
+      </>
+    );
+  };
+
+  const getCertDetails = () => {
+    return (
+      <>
+        {certs.length > 0 &&
+          certs.map((cert, index) => (
+            <StackItem key={index}>{formatCertInfo(cert, index)}</StackItem>
+          ))}
+      </>
+    );
+  };
```

Comment:
Same as for `getExtensionsList` this must be a react component since it is returning JSX:

```typescript
  const getCertDetails = () => {
    return (
      <>
        {certs.length > 0 &&
          certs.map((cert, index) => (
            <StackItem key={index}>{formatCertInfo(cert, index)}</StackItem>
          ))}
      </>
    );
  };
```